### DISCUSSION TITLE
How to adjust the parameters of motion_generation as needed (lack of relevant parameter descriptions)

### DISCUSSION THREAD

**Discussion by pmj110119** [NVlabs/curobo#99] [Software Q&A] [answered]
I am currently using the default configuration parameters in the example, and I found that it will generate a fixed trajectory consisting of 32 points.
```python
motion_gen_config = MotionGenConfig.load_from_robot_config(
            self.robot_cfg,
            self.world_cfg,
            self.tensor_args,
            trajopt_tsteps=32,
            collision_checker_type=CollisionCheckerType.BLOX,
            use_cuda_graph=True,
            num_trajopt_seeds=24,
            num_graph_seeds=24,
            interpolation_dt=0.03,  
            collision_activation_distance=0.025,
            acceleration_scale=1.0,
            self_collision_check=False,     
            maximum_trajectory_dt=0.25, 
            finetune_dt_scale=1.05,
            fixed_iters_trajopt=True,
            finetune_trajopt_iters=300,
            minimize_jerk=True,
        )
```

I originally thought that `interpolation_dt=0.03` meant to use interpolation to obtain denser trajectory points at 30ms intervals, but calling `get_interpolated_plan` did not achieve the expected effect. What is the meaning of this part of parameters? How can I adjust it to get higher frequency traces.

(Regardless of the code or documentation, there seems to be no relevant explanation for this part of the parameters. If your team will supplement the documentation later, I believe it will significantly improve the user experience!)

**Comment by balakumar-s** [comment, marked as answer]
`get_interpolated_plan()` will give a trajectory that is interpolated at the dt specified by `interpolation_dt`. Note that this trajectory will be of varying length as shorter trajectories will have less number of waypoints (at 0.03 dt steps) than longer trajectories. 

We are currently working on adding features to make the library more useful. Once we have added features, we will also add documentation to the code.

**Reply by pmj110119** [reply]
Thanks for reply!! Look forward to your update.
